webapp/src/shared/components/Row/Row.tsx

```diff
@@ -0,0 +1,54 @@
+import Icon, { IconVariant, IconSize } from '../Icon/Icon';
+import './Row.css';
+import { SmallAvatar, AvatarProps } from '../Avatar/Avatar';
+import { TextVariant, TextWeight } from '../Text/Text';
+import { default as Text } from '../Text/Text';
+
+type RowProps = {
+  iconVariant?: IconVariant;
+  onClick?: () => void;
+  avProps?: AvatarProps;
+  children?: string[];
+};
+
+export default function Row({
+  iconVariant,
+  onClick,
+  avProps,
+  children,
+}: RowProps) {
+  return (
+    <button className={`row paragraph-regular`} onClick={onClick}>
+      <div className={`row_wrapper`}>
+        {avProps && <SmallAvatar {...avProps} />}
+        {children && (
+          <div className={`row_text_wrapper`}>
+            {children[0] && (
```

Comment:
Same as before, `children[0]` is not very legibile :(  Apart from this, for me, an array should be a data structure to use when all components had the same relevance. Here the position `0` have a different relevance from `1`, they represent different things (title and subtitle) and we should take that into account.